### PR TITLE
[Task-14][feat] Adding cli functionality

### PR DESCRIPTION
**Pull request by OlegPanasyuk**
-	Rebiuld structure of folders

-	Adding ability to work with sequelize-cli

- 	Adding mock data to courses and users

-	Adding nom for reading excel files

### CODE REVIEW

File: database/v2/config/config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "username": "uxhzjohnwjgzlj",
+        "password": "efeaf8c1794fcc1124c20d09f76fc95f081e60a5d109946d9c3ab2b4114d7671",
+        "database": "d30fvv1srb6p0d",
+        "host": "ec2-54-75-235-28.eu-west-1.compute.amazonaws.com",
+        "dialect": "postgres",
+        "dialectOptions": {
+          "ssl": true,
+        },
+        "logging": false,
+    }
```
Comment (slavasgod) [inline database/v2/config/config.js]: This is a seccure data, don't commit it. We have these settings in .env file

File: database/v2/seeders/20191021130637-demo-user.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+'use strict';
+const hashHelpers = require('../../../api/helpers/hashHelpers');
```
Comment (slavasgod) [inline database/v2/seeders/20191021130637-demo-user.js:1]: Why are you using use strict?

File: database/v2/seeders/20191021130637-demo-user.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    const data = await hashHelpers.createHash("password");
+    const obj = {
+      userName: `John Doe`,
+      email: `demo${i}@demo.com`,
+      password: data,
+      role: 'User',
+      locale: 'ru',
+      resetPasswordToken: null,
+      createdAt: new Date(),
+      updatedAt: new Date()
```
Comment (slavasgod) [inline database/v2/seeders/20191021130637-demo-user.js]: These fields have default value, you can skip them

File: database/v2/seeders/20191021130637-demo-user.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    };
+    return obj;
+}
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    
+    const dataTemp = await Promise.all(Array(10).fill(1).map((el, i) => {
+      return createMockUser(i).then(obj => obj);
```
Comment (slavasgod) [inline database/v2/seeders/20191021130637-demo-user.js]: Use meaningful variables in array methods: el => user, i => index.
Replace obj to smth else.

File: database/v2/seeders/20191022152204-demo-courses.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
+'use strict';
+const readXlsxFile = require('read-excel-file/node');
+
+async function getMockData() {
+  let rows = await readXlsxFile('../v2/asserts/mock-courses.xlsx');
+  let data = [];
+  for (let i = 2; i < 17; i++) {
```
Comment (slavasgod) [inline database/v2/seeders/20191022152204-demo-courses.js]: Use const instead of let

File: database/v2/seeders/20191022152204-demo-courses.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    let el = rows[i];
+    data.push({
```
Comment (slavasgod) [inline database/v2/seeders/20191022152204-demo-courses.js:8]: Replace el to meaningful variable

File: database/v2/seeders/20191021130637-demo-user.js (hunk 2 above)
Comment (OlegPanasyuk) [reply database/v2/seeders/20191021130637-demo-user.js:1]: Done

File: database/v2/seeders/20191021130637-demo-user.js (hunk 3 above)
Comment (OlegPanasyuk) [reply database/v2/seeders/20191021130637-demo-user.js]: Done

File: database/v2/seeders/20191021130637-demo-user.js (hunk 4 above)
Comment (OlegPanasyuk) [reply database/v2/seeders/20191021130637-demo-user.js]: Done

File: database/v2/seeders/20191022152204-demo-courses.js (hunk 5 above)
Comment (OlegPanasyuk) [reply database/v2/seeders/20191022152204-demo-courses.js]: Done

File: database/v2/seeders/20191022152204-demo-courses.js (hunk 6 above)
Comment (OlegPanasyuk) [reply database/v2/seeders/20191022152204-demo-courses.js:8]: done

File: database/v2/config/config.js (hunk 1 above)
Comment (OlegPanasyuk) [reply database/v2/config/config.js]: Done

**Review by slavasgod — APPROVED** [review]

**Review by melonholy — APPROVED** [review]

**Pull request merged**